docs/plaindate.md

```diff
@@ -129,14 +129,14 @@ date = Temporal.PlainDate.from({ year: 2001, month: 1, day: 32 }, { overflow: 'r
 Compares two `Temporal.PlainDate` objects.
 Returns an integer indicating whether `one` comes before or after or is equal to `two`.
 
-- &minus;1 if `one` comes before `two`;
-- 0 if `one` and `two` are the same date and their `calendar` properties are also the same;
-- 1 if `one` comes after `two`.
+- &minus;1 if `one` comes before `two`
+- 0 if `one` and `two` are the same real-world date, regardless of calendar
```

Comment:
The use of 'real-world' here and elsewhere feels slightly off; if anything, the calendar dates are more real-world because that's what people use. It could feel somewhat like we are implying there's an underlying truth that human calendars are obscuring.

I'm not sure how else to phrase it though. Maybe "same ISO date", but that isn't very clear to someone who doesn't know what an ISO date is. I also considered "same 24-hour period" but that's not accurate either, the same date can refer to a differently staggered 24-hour period in different time zones. Maybe "same midnight-to-midnight period" (ugh)?

Reply:
Yeah your thought process is just like the one I went through trying to name this darn thing. Hard bikeshed. Maybe "same date in the ISO 8601 calendar" might be wordier but clearest?

Reply:
I agree, that sounds like the least bad wording so far!

Reply:
We didn't hear any other feedback, so I just adjusted the text to match that "least bad" wording. Also rebased. Should be ready to merge.

Reply:
Great, let's do it!